src/app/productConfig/productConfigForm/productConfigForm.component.js

```diff
@@ -351,6 +386,7 @@ export default angular
       isEdit: '<',
       uri: '<',
       defaultFrequency: '<',
+      isBrandedCheckout: '<',
```

Comment:
You can use `envService.read('isBrandedCheckout')` instead of having to pass it around.